website/project/views/node.py

```diff
@@ -277,6 +277,17 @@ def node_setting(auth, node, **kwargs):
         'level': node.comment_level,
     }
 
+    addon_settings = {}
+    for addon in ['forward']:
```

Comment:
Nitpick: The loop with a 1-item list is a bit awkward. I think it can be simplified to:

```python
forward_config = apps.get_app_config('addons_forward')
settings_dict = forward_config.to_json()
# ...
addon_settings = {
    'forward': settings_dict,
}
```

Reply:
I agree in this case - we have only one add-on that has a configuration in the project settings (forward add-on). But if someone has more, he/she will add more elements to the loop. 
In our fork of your project we add more such add-ons, so the loop is justifiable.